src/scraper/scraper.cpp

```diff
@@ -67,10 +67,18 @@ struct ScraperFileManifest
     int64_t timestamp;
 };
 
+// Both TeamIDMap and ProjTeamETags are protected by cs_TeamIDMap.
 // --------------- project -------------team name -- teamID
 typedef std::map<std::string, std::map<std::string, int64_t>> mTeamIDs;
 mTeamIDs TeamIDMap;
 
+// ProjTeamETags is not persisted to disk. There would be little to be gained by doing so. The scrapers are restarted very
+// rarely, and on restart, this would only save downloading team files for those projects that have one or TeamIDs missing AND
+// an ETag had NOT changed since the last pull. Not worth the complexity.
+// --------------- project ---- eTag
+typedef std::map<std::string, std::string> mProjectTeamETags;
+mProjectTeamETags ProjTeamETags;
```

Comment:
Might want to come back around later and swap the casing on `mTeamIDs` &lrarr; `TeamIDMap` and `mProjectTeamETags` &lrarr; `ProjTeamETags` to match the convention so that the type name is Pascal case and the variable is camel case. My tired brain glitched for a few seconds looking at these 🙂

Reply:
Hehe.